src/components/Dropdown.js

```diff
@@ -80,29 +85,60 @@ export default class Dropdown extends React.Component {
     }
   }
 
+  handleKeys = event => {
+    if (this.refs.list) this.refs.list.handleKeys(event)
+  }
+
+  chooseChild = choice => {
+    const link = findChildLink(findDOMNode(this.refs[choice.ref]))
```

Comment:
No, choice is an Element, not a Dom node. The stuff where I clone and add refs on line 115, and then call findDOMNode is the only way to get a node out of an Element.